hazelcast/src/test/java/com/hazelcast/map/nearcache/NearCacheTest.java

```diff
@@ -117,11 +117,18 @@ public void testBasicUsage() {
         }
 
         map.clear();
-        for (HazelcastInstance instance : instances) {
-            NearCache nearCache = getNearCache(mapName, instance);
-            int size = nearCache.size();
-            assertEquals("Near Cache size should be 0 but was " + size, 0, size);
-        }
+
+        assertTrueEventually(new AssertTask() {
```

Comment:
Yes, but it was not needed before, was it? And that was my point. This looks like a behavioral change to me, so we need at least to document this in the release notes.